influxql/query_manager.go

```diff
@@ -222,7 +222,7 @@ func (qm *defaultQueryManager) Query(qid uint64) (*queryTask, bool) {
 	return query, ok
 }
 
-func (qm *defaultQueryManager) waitForQuery(qid uint64, timeout time.Duration, closing <-chan struct{}, monitorCh <-chan error) {
+func (qm *defaultQueryManager) waitForQuery(qid uint64, interrupt <-chan struct{}, timeout time.Duration, closing <-chan struct{}, monitorCh <-chan error) {
 	var timer <-chan time.Time
 	if timeout != 0 {
 		timer = time.After(timeout)
```

Comment:
nit: Do you want to remove the `time.After` as well based on our discussion this morning?